src/connection/adapter/base_http_adapter.ts

```diff
@@ -294,17 +294,30 @@ export abstract class BaseHttpAdapter implements Connection {
   }
 
   private logResponse(
+    request: Http.ClientRequest,
     params: RequestParams,
     response: Http.IncomingMessage,
     startTimestamp: number
   ) {
+    const getRequestHeaders = () => {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { authorization, ...headers } = request.getHeaders()
+      return headers
+    }
+    const getRequestBody = () => {
+      return typeof params.body === 'string' ? params.body : 'Stream'
+    }
     const duration = Date.now() - startTimestamp
-
-    this.logger.debug(
-      `[http adapter] response: ${params.method} ${params.url.pathname}${
-        params.url.search ? ` ${params.url.search}` : ''
-      } ${response.statusCode} ${duration}ms`
-    )
+    this.logger.debug('[HTTP Adapter] Got a response from ClickHouse', {
+      request_method: params.method,
+      request_path: params.url.pathname,
```

Comment:
We can keep both, but in the case of CH, request_method and request_path are almost always `POST` and `/` ( for `ping` method they are `GET` and `/ping`)